samples/correlation-csharp/.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
We can probably drop most of the .git* files, as we have global ones for the repo.

Reply:
This file should have been deleted. If you resolve this issue, I believe it might go away from the UI?